Modules/Core/Tests/Feature/ImportControllerTest.php

```
@@ -35,4 +37,368 @@ public function it_displays_import_page(): void
+        $formData = [
+            'btn_submit' => '1',
+            'files' => [],
+        ];
+
+        /** Act */
+        $this->actingAs($user);
+        $response = $this->post(route('import.form'), $formData);
+
+        /** Assert */
+        $response->assertRedirect(route('import.index'));
+    }
```

PHPMD (2.15.0)

The method it_handles_import_with_no_files_selected is not named in camelCase. (undefined)

(CamelCaseMethodName)